.npmrc

```diff
@@ -0,0 +1 @@
+save-exact = true
```

Comment:
Ensures that on adding new `package.json` files, it saves the exact version, not something with a caret like `^3.5.3`. 

This should help ensure our builds are the same, even in our different locals.